pkg/services/notificator/deps.go

```diff
@@ -0,0 +1,19 @@
+package notificator
+
+import objectSDKAddress "github.com/nspcc-dev/neofs-sdk-go/object/address"
+
+// NotificationSource provides notification that should
+// be written via the NotificationWriter.
+type NotificationSource interface {
+	// Iterate must iterate over all notification for the
+	// provided epoch and call handler for all of them.
+	Iterate(epoch uint64, handler func(topic string, addr *objectSDKAddress.Address))
+}
+
+// NotificationWriter notifies all the subscribers
+// about a new object notifications.
+type NotificationWriter interface {
+	// Notify must send string representation of the object
```

Comment:
Strange to see `must send string representation`, why `address` is not a `string` then?

Reply:
it uses part of oid to make "exactly once delivery". did not know if this should be noted in the interface description but Im sure that this part should not be parsed in the string representation